frontend/packages/dev-console/src/components/topology/filters/__tests__/FilterDropdown.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { SelectOption } from '@patternfly/react-core';
+import FilterDropdown from '../FilterDropdown';
+import { DisplayFilters } from '../filter-utils';
+
+const VALID_FILTERS = {
+  podCount: true,
+  eventSources: true,
+  knativeServices: true,
+  appGrouping: true,
+  operatorGrouping: true,
+};
+
+describe(FilterDropdown.displayName, () => {
+  let dropdownFilter: DisplayFilters;
+  let onChange: () => void;
+  beforeEach(() => {
+    dropdownFilter = { ...VALID_FILTERS };
+    onChange = jasmine.createSpy();
+  });
+
+  it('should exists', () => {
+    const wrapper = shallow(<FilterDropdown filters={dropdownFilter} onChange={onChange} />);
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should have 5 filters in total', () => {
```

Comment:
We shouldn't have a number in the test description... it's easy for this to get out of sync (see my previous review comment to see what I named it)